src/css/editor.css.js

```diff
@@ -583,4 +583,9 @@ styles.join("\\n")
     white-space: pre;
     opacity: 0.7;
     margin: 0 10px;
+}
+
+.ace_ghost_text {
+    opacity: 0.5;
+    font-style: italic;
```

Comment:
Chrome console doesn't use italic, which makes it look more like the part of the typed word. I think it is better but maybe it's just a matter of getting used to.